viadot/tasks/sap_bw.py

```diff
@@ -0,0 +1,122 @@
+import pandas as pd
+
+
+from typing import List, Dict, Any, Literal
+from prefect import Task
+from prefect.tasks.secrets import PrefectSecret
+from prefect.utilities import logging
+from viadot.exceptions import ValidationError
+from viadot.sources import SAPBW
+from viadot.task_utils import *
+
+logger = logging.get_logger()
+
+
+class SAPBWToDF(Task):
+    def __init__(
+        self,
+        sapbw_credentials: dict = None,
+        sapbw_credentials_key: str = "SAP",
+        env: str = "BW",
+        *args,
+        **kwargs,
+    ):
+        """
+        A task for quering the SAP BW (SAP Business Warehouse) source using pyrfc library.
+
+        Args:
+            sapbw_credentials (dict, optional): Credentials to SAP BW server. Defaults to None.
+            sapbw_credentials_key (str, optional): Azure KV secret. Defaults to "SAP".
+            env (str, optional): SAP environment. Defaults to "BW".
+        """
+        if sapbw_credentials is None:
+            self.sapbw_credentials = credentials_loader.run(
+                credentials_secret=sapbw_credentials_key
+            )
+            self.sapbw_credentials = self.sapbw_credentials[env]
```

Comment:
Please check if this option works. You can remove 1 line by adding `get()` to the end of the first one.
```suggestion
            self.sapbw_credentials = credentials_loader.run(
                credentials_secret=sapbw_credentials_key
            ).get(env)
```

Reply:
changed